crates/query-engine/translation/src/translation/query/root.rs

```diff
@@ -160,8 +160,6 @@ fn translate_query_part(
 
     select.joins.extend(relationship_joins);
 
-    select.joins.extend(filter_joins);
```

Comment:
This is gone because the joins are happening where?

Reply:
Oh - I think I just found the answer: https://github.com/hasura/ndc-postgres/pull/463/files#r1602699713